legal-api/tests/unit/api/test_business_filings/test_filings.py

```diff
@@ -785,20 +787,37 @@ def test_get_correct_fee_codes(session):
     for director in free_cod['filing']['changeOfDirectors']['directors']:
         if not all(action in ['nameChanged', 'addressChanged'] for action in director.get('actions', [])):
             director['actions'] = ['nameChanged', 'addressChanged']
+    bc_coa = copy.deepcopy(FILING_HEADER)
+    bc_coa['filing']['header']['name'] = 'changeOfAddress'
+    bc_coa['filing']['business']['legalType'] = 'BC'
+    bc_coa['filing']['changeOfAddress'] = CHANGE_OF_ADDRESS
+    bc_cod = copy.deepcopy(FILING_HEADER)
+    cod['filing']['header']['name'] = 'changeOfDirectors'
```

Comment:
duplicate of line 779, should it be bc_cod?

Reply:
It should indeed! Updated in next commit